RNTester/README.md

```diff
@@ -8,21 +8,15 @@ Before running the app, make sure you ran:
 
     # in this folder
     npm install
-    # and in the toplevel folder
-    cd ..
-    npm install
 
 ### Running on iOS
 
 Mac OS and Xcode are required.
 
-- Start the packager with `npm start` **[important]**
 - Open a new tab in your terminal and start the bucklescript watcher with `npm run watch`
```

Comment:
sure we could build it initially, but if the people don't run the watcher, no changes will have an effect